src/stdx/traits.hpp

```diff
@@ -33,4 +33,9 @@ struct priority_tag_t : priority_tag_t<N - 1> {};
 template <std::size_t N>
 inline constexpr priority_tag_t<N> priority_tag{};
 
+template <class T, class U>
+struct specialize_same_template : std::is_same<T, U> {};
```

Comment:
what about renaming this to
`(is_)same_template_specialization`

Reply:
and is this used?

Reply:
Removed, not used